client/my-sites/themes/theme-showcase.jsx

```diff
@@ -14,7 +14,38 @@ import Main from 'components/main';
 import ThemePreview from './theme-preview';
 import ThemesSelection from './themes-selection';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
-import { addTracking } from './helpers';
+import {
+	getDetailsUrl,
+	getSupportUrl,
+	getHelpUrl,
+	isPremium,
+	addTracking
+} from './helpers';
+
+export const sheetOptions = ( site = false, isJetpack = false ) => ( {
```

Comment:
❤️ it! Great idea. Super tiny suggestion: give the function name a verb, like `getSheetOptions()`